### DISCUSSION TITLE
Fake 29EE512, 512 Kbit Memory Cards

### DISCUSSION THREAD

**Discussion by ghost** [brad-lin/FreePSXBoot#88] [General]
I got a few of the fake 29EE512 512Kb Memory Cards and it took a few tries while stumbling blindly ( I don't really know what I'm doing )  until I managed to make it work.  I used a hex editor to split the lines in two and duplicated the first half. Thanks!

**Comment by danielw86dev** [comment]
I confirm last builder (768069e) work with fake 512kb, i build image with tonyhax and card is booting and working. Very thanks!

What im do:

Compile last builder
make image slot 2 with payload tonyhax 1.4.3
extract unirom 8.0.I, replace mcd for my BIOS 7502, biild ISO with mkpsxiso
boot with swap disk metod and install freepsxboot to menory card (fake 512kb)
